.devcontainer/Dockerfile

```diff
@@ -0,0 +1,67 @@
+ARG RUST_VERSION=1.68.2
+ARG CMAKE_VERSION=3.26.3
```

Comment:
Do we need to specify the version here?

Rustup will automatically download the version specified in rust-toolchain.toml so the version here is just used for bootstrap.

For cmake, the minimum supported version is 3.19.0. We can probably just use the package provided by the base distro image (debian-11 in this case).

Reply:
722f18b
+ Removed CMake version, using Debian default
+ Synced Rust version with MSVC version to save users time. Users will no longer need to redownload Rust after opening the devcontainer.